biomes/land/mountains-small/temperate/semi-humid/highlands.yml

```diff
@@ -0,0 +1,34 @@
+id: HIGHLANDS
+type: BIOME
+extends: [ EQ_ERODED_TERRACED_MOUNTAINS, CARVING_LAND, BASE ]
+vanilla: minecraft:plains
+color: $biomes/colors.yml:HIGHLANDS
+
```

Comment:
Again, for future reference, assign the biome its own colors, I recommend going [here](https://minecraft.fandom.com/wiki/Color) and starting from base values from whatever biome you choose. (This is future-proofing for if/when the "vanilla" key is ever dropped.